Dockerfile.template

```diff
@@ -124,7 +125,20 @@ RUN set -ex; \
 	\
 	apt-mark auto '.*' > /dev/null; \
 	[ -z "$savedAptMark" ] || apt-mark manual $savedAptMark; \
-	apt-get purge -y --auto-remove -o APT::AutoRemove::RecommendsImportant=false
+	apt-get purge -y --auto-remove -o APT::AutoRemove::RecommendsImportant=false; \
+	echo {{
+		{
+			name: "gcc",
+			version: .version,
+			params: {
+				os_name: "debian",
+				os_version: .debian.version
+			},
+			licenses: [
+				"Apache-2.0"
```

Comment:
Is `gcc` itself under the Apache license?  I would've guessed GPL?

(This is one of my biggest concerns with us maintaining this metadata manually :see_no_evil:)

Reply:
Definitely GPL: https://gcc.gnu.org/git/?p=gcc.git;a=blob;f=COPYING;h=623b6258a134210f0b0ada106fdaab7f0370d9c5;hb=refs/heads/master

Reply:
Nice catch!